projects/wpdx/src/app/rehab-prio/rehab-prio.component.html

```diff
@@ -264,7 +264,7 @@
                     <mat-button-toggle-group [(ngModel)]="rpState.nc_limit" aria-label="ADM Level" appearance='Legacy'>
                         <mat-button-toggle value="50" [disableRipple]='true'>50</mat-button-toggle>
```

Comment:
If you write here `[value]='50'` instead, you will get a number and not a string in `nc_limit`